src/components/molecules/Visualizer/Widget/index.tsx

```diff
@@ -73,8 +73,8 @@ export default function WidgetComponent<PP = any, SP = any>({
 
   if (!w) return null;
 
-  const Builtin =
-    w?.pluginId && w.extensionId ? builtin[`${w.pluginId}/${w.extensionId}`] : undefined;
+  const builtinWidgetId = `${w.pluginId}/${w.extensionId}`;
```

Comment:
This is't necessarily a built-in widget's ID. 
```suggestion
  const widgetId = `${w.pluginId}/${w.extensionId}`;
```